### PR TITLE
Document workaround to out of space

### PR DESCRIPTION
**Pull request by adamtheturtle**
Perhaps the explanation here is a bit vague - I don't have a full understanding myself. However, the information here would have saved me a few hours so I believe that this is worth merging.

See https://jira.mesosphere.com/browse/DCOS-16466.

### CODE REVIEW

**Comment by adamtheturtle** [comment]
cc @mhrabovcin @karlkfi

**Comment by karlkfi** [comment]
This might be related to https://jira.mesosphere.com/browse/DCOS_OSS-1240 which is why a bunch of our integration tests are breaking.

**Comment by adamtheturtle** [comment]
@karlkfi what would you like me to do with this PR?

**Comment by jgehrcke** [comment]
As this would affect non-DC/OS-related Docker workflows on the same machine, let's add a warning and then merge.

**Comment by karlkfi** [comment]
This advice seems rather nuclear. What was causing you to run out of disk space?

**Comment by adamtheturtle** [comment]
@karlkfi As per the PR description I have no idea.

**Comment by adamtheturtle** [comment]
I imagine that there could be a better solution but I don't know of it.

**Comment by adamtheturtle** [comment]
@karlkfi I guess the question is would you rather have this "nuclear" option documented or not. If someone in the future has this issue and chooses to investigate further they can always remove these docs. Personally I think that it is better to have them than not.

**Comment by karlkfi** [comment]
I'm not excited about having a big log hunk in the readme and a mac-only fix. 
How do you feel about having a troubleshooting faq like dcos-vagrant's?
https://github.com/dcos/dcos-vagrant/blob/master/docs/troubleshooting.md

**Comment by adamtheturtle** [comment]
@karlkfi I think that having a separate Troubleshooting doc would be good and I will create that as part of this PR.

**Comment by adamtheturtle** [comment]
@karlkfi I have:

* Created a new troubleshooting document
* Moved existing troubleshooting information there
* Made some changes to existing troubleshooting information
* Linked to the new document from `README.md`.

Therefore I consider this ready for another review.

File: troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: Traceback (most recent call last):
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: File "/opt/mesosphere/packages/exhibitor--72d9d8f947e5411eda524d40dde1a58edeb158ed/usr/exhibitor/start_exhibitor.py", l
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: """)
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: File "/opt/mesosphere/packages/exhibitor--72d9d8f947e5411eda524d40dde1a58edeb158ed/usr/exhibitor/start_exhibitor.py", l
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: f.write(contents)
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: OSError: [Errno 28] No space left on device
+```
+
+On Docker for Mac, this can be fixed by deleting the disk image.
```
Comment (jgehrcke) [inline troubleshooting.md]: Should we add a warning here that this obliterates all locally stored Docker images, including those unrelated to DC/OS (I don't know anything about Docker on a Mac, but this is what I believe to have understood from here...)?

Comment (adamtheturtle) [inline troubleshooting.md]: Yes, I will document this.

**Comment by adamtheturtle** [comment]
@jgehrcke @karlkfi I have now addressed all comments so I think that this can be merged.

**Comment by jgehrcke** [comment]
Thanks!

**Pull request merged**